superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -545,14 +552,24 @@ function ChartList(props: ChartListProps) {
   }
   if (isFeatureEnabled(FeatureFlag.VERSIONED_EXPORT)) {
     subMenuButtons.push({
-      name: <Icons.Import />,
+      name: (
+        <Tooltip
+          id="import-tooltip"
+          title={t('Import Chart')}
+          placement="bottomRight"
+        >
+          <Icons.Import />
+        </Tooltip>
+      ),
       buttonStyle: 'link',
       onClick: openChartImportModal,
     });
   }
   return (
     <>
-      <SubMenu name={t('Charts')} buttons={subMenuButtons} />
+      <StyledNavbarRight>
```

Comment:
Does it make sense to just apply these styles to the SubMenu? Seems like the default behavior/expectation is that these items are centered.

Reply:
Good call, that's a lot cleaner! Thank you!